internal/manifests/opampbridge/deployment.go

```diff
@@ -28,6 +28,9 @@ import (
 func Deployment(params manifests.Params) *appsv1.Deployment {
 	name := naming.OpAMPBridge(params.OpAMPBridge.Name)
 	labels := manifestutils.Labels(params.OpAMPBridge.ObjectMeta, name, params.OpAMPBridge.Spec.Image, ComponentOpAMPBridge, params.Config.LabelsFilter())
+	if getDNSPolicy(params.OpAMPBridge) == "None" && params.OpAMPBridge.Spec.PodDNSConfig.Nameservers == nil {
```

Comment:
for all of these calls we should be using the upstream corev1 type rather than string matching given we return the coreV1 type

Reply:
Thanks @jaronoff97! I will update it accordingly.